generators/server/templates/build.gradle.ejs

```diff
@@ -290,7 +290,7 @@ dependencies {
         exclude group: "org.springframework", module: "spring-webmvc"
     }<% } %>
     <%_ // This is useful for major Spring Boot migration
-    if (!SPRING_BOOT_VERSION.endsWith('RELEASE')) { _%>
+    if (!SPRING_BOOT_VERSION.endsWith('RELEASE') || SPRING_BOOT_VERSION.startsWith('2.4')) { _%>
```

Comment:
Is it always mandatory

Reply:
We already had issues with it (https://github.com/jhipster/generator-jhipster/pull/12368#issuecomment-685700956) so we only include it in development versions, not in releases.